atomic

```diff
@@ -683,6 +682,12 @@ def create_parser(help_text):
     verifyp.add_argument("-v", "--verbose", default=False,
                           action="store_true",
                           help=_("Report status of each layer"))
+    verifyp.add_argument("-V", "--validate", default=True,
+                                action="store_false",
+                                help=_("disable validating system images"))
+    verifyp.add_argument("-g", "--generate", default=False,
```

Comment:
It seems weird to have this option under `verify`.  I'd expect it to be a system-wide setting one can turn on by default.

Also...I'm a bit concerned about the performance of all of this.  Our intent with OSTree BTW is to drive immutability down into the storage stack - then you don't really need verify since the files are truly immutable.  See discussion in [http://www.spinics.net/lists/linux-fsdevel/msg75085.html](O_OBJECT).